roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,32 @@
   with_items: network_interfaces_d
   tags: ['prereboot']
 
+- name: set slrouter address
+  set_fact: sl_router_address="{{ ansible_default_ipv4.network.split('.')[0] }}.{{ ansible_default_ipv4.network.split('.')[1] }}.{{ansible_default_ipv4.network.split('.')[2] }}.{{ (ansible_default_ipv4.network.split('.')[3] | int) + 1 }}"
+  when: ansible_default_ipv4.interface == hostvars[inventory_hostname][primary_interface].device   
+
+- name: add static route for multi subnet
+  command: "/sbin/ip route replace {{ item }} via {{ sl_router_address }}"
+  when: (not (ansible_default_ipv4.network ==  ( item | ipaddr('network') ))) and (ansible_default_ipv4.interface == hostvars[inventory_hostname][primary_interface].device)
+  with_items: "{{ undercloud_cidr }}"
```

Comment:
It does feel awkward to only call out one router to add here. What if we needed multiple routes for multiple network segments? Should we just iterate through all the routes to add them in case they were missing?